frontend/js/src/common/brainzplayer/AppleMusicPlayer.tsx

```diff
@@ -173,10 +184,15 @@ export default class AppleMusicPlayer
       return;
     }
     try {
-      await this.appleMusicPlayer.setQueue({
+      const queueData = await this.appleMusicPlayer.setQueue({
         song: appleMusicId,
         startPlaying: true,
       });
+      const albumId =
+        // eslint-disable-next-line no-underscore-dangle
+        queueData._queueItems[0].item.relationships.albums.data[0].id;
```

Comment:
That's looks a bit unsafe. If any element of this long chain doesn't exist or is null/undefined, it will throw an error.
is this the failure condition we want in this case?

Also not sure about accessing _underscore shadow variables, are you missing a getter or something?